src/content/docs/en/reference/experimental-flags/fonts.mdx

```diff
@@ -0,0 +1,682 @@
+
+<Steps>
+
+1. Register an empty family to start with:
+
+    ```js title="astro.config.mjs" ins={4-8}
+    import { defineConfig } from "astro/config"
+
+    export default defineConfig({
+        experimental: {
+            fonts: [{
+                // We are going to add things here
+            }]
+        }
+    })
+    ```
+
+2. Pick a provider from the [built-in remote provider](#available-remote-font-providers) or the [local provider](#local-font-families). In this example, we'll use the Google font provider:
+
+    ```js title="astro.config.mjs" del={1} ins={2,7}
+    import { defineConfig } from "astro/config"
+    import { defineConfig, fontProviders } from "astro/config"
+
+    export default defineConfig({
+        experimental: {
+            fonts: [{
+                provider: fontProviders.google(),
+            }]
+        }
+    })
+    ```
+
+3. Pick a font family supported by your provider (documented for each provider in the [reference](#available-remote-font-providers)). For our example, choose one on the [Google Fonts website](https://fonts.google.com/):
+
+    ```js title="astro.config.mjs" ins={7}
+    import { defineConfig, fontProviders } from "astro/config"
+
+    export default defineConfig({
+        experimental: {
+            fonts: [{
+                provider: fontProviders.google(),
+                name: "Roboto",
+            }]
+        }
+    })
+    ```
+
+3. Choose a CSS variable. This will allow you to reference the font family you just registered in CSS:
+
+    ```js title="astro.config.mjs" ins={8}
+    import { defineConfig, fontProviders } from "astro/config"
+
+    export default defineConfig({
+        experimental: {
+            fonts: [{
+                provider: fontProviders.google(),
+                name: "Roboto",
+                cssVariable: "--font-roboto",
+            }]
+        }
+    })
+    ```
+
+</Steps>
+
+More options are available and depend on your provider, refer to the [reference](#reference) to learn more.
+
+### `<Font />` component
+
+This component outputs style tags and preload links for a given font family:
+
+<Steps>
+
+1. Import the component to be used in your page `<head>`:
+
+    ```astro title="src/components/Head.astro" ins={2}
+    ---
+    import { Font } from 'astro:assets'
+    ---
+    ```
+
+2. Call the component and specify a `family`. It takes the value of the `cssVariable` in your Astro configuration:
+
+    ```astro title="src/components/Head.astro" ins={5}
+    ---
+    import { Font } from 'astro:assets'
+    ---
+
+    <Font family="--font-roboto" preload />
+    ```
+
+    `family` is strongly typed using the values from your Astro configuration to help you.
+
+</Steps>
+
+### Use the CSS variable
+
+Now that the `<Font />` component generates CSS with font declarations, you can reference the font family using the `cssVariable` you used in your Astro configuration:
```

Comment:
```suggestion
Since the `<Font />` component generates CSS with font declarations, you can reference the font family using the `cssVariable` defined in your Astro configuration:
```

Since it's a new feature it felt weird saying it as if this is a change from a previous version